src/bttrx_control.cpp

```diff
@@ -274,6 +290,25 @@ string BTTRX_CONTROL::ParameterTypeToString(ParameterType parameter_type)
 	return return_value;
 }
 
+/**
+ * @brief Set Callsign
+ * 
+ * @param callsign 
+ * @return ResultType 
+ */
+ResultType BTTRX_CONTROL::handleSetCallsign(string callsign)
+{
+	if (callsign != "") {
+		serial_->dbg_println("Set Callsign to: " + callsign);
+	} else {
+		serial_->dbg_println("Callsign deleted");
+	}
+
```

Comment:
Please add a check for the maximum string length of 6 characters.

Reply:
[x] done